Makefile.gh

```diff
@@ -69,6 +69,9 @@ build-wheel: pip
 			cd -;\
 		fi;\
 	done
+	ls
+	ls PYPI_UPLOAD
+	twine check ./PYPI_UPLOAD/*.{tar.gz,whl}
```

Comment:
Looks like the shell is not happy with this glob.  It's OK to use just a '*', as can be seen here:

https://github.com/clebergnu/avocado/actions/runs/6511560550/job/17687503910#step:4:2837